feat(navbar): Added a link to project support in help center (#543)
ethereum-optimism/community-hub: src/docs/biz/README.md

The OPLabs team gets a flurry of inbound messages and calls every day and we reply to them as efficiently as possible, but we are only human! 
Please be patient if you are in need of direct support. 
We cannot always provide direct help, but we strive to help everyone succeed on Optimism through the various resources we provide to projects. 

## Developer Support 🙋💻

Steps to take if you would like developer support immediately, ranked in order of response times. 

1. See our [Developer Documentation](../developers/), [Tutorials](https://github.com/ethereum-optimism/optimism-tutorial), and [Help Center](https://help.optimism.io)
1. Head over to [**#dev-support**](https://discord.com/channels/667044843901681675/887914409207414785) on discord for the fastest help 
1. Join [this TG group](https://t.me/+Cb7q0a1YqItkZTRk) and ask for help there
1. Last attempt: [Fill out this form](https://optimismpbc.typeform.com/get-in-touch?typeform-source=community.optimism.io)

## Marketing Requests 🦸🦸

> When your project is deployed on Optimism, you’ll be added to [our ecosystem page](https://www.optimism.io/apps/all). 

Reviewer: ```suggestion
When your project is deployed on Optimism, you can be added to [our ecosystem page](https://www.optimism.io/apps/all). 
```

> Simply [fill out this form](https://optimismpbc.typeform.com/op-marketing) to be included. 

Reviewer: ```suggestion
Simply [fill out this form](https://optimismpbc.typeform.com/op-marketing) to be included. Inclusion is at our discretion. 
```

> We add new projects [here](https://www.optimism.io/apps/all) every week, but you need to deploy first. 

Reviewer: ```suggestion
We add new projects live on the Optimism mainnet [here](https://www.optimism.io/apps/all) once a week.
```

### Promoting Values before anything else

> We love to promote values-aligned projects as part of our broader content program.

Reviewer: Do we have our values anywhere publicly?

### Tweeting

We do the occasional retweet but the bar for these is kept quite high in order to keep our Twitter feed as valuable/high signal as possible. 
Other than RT's, we may like or reply to a tweet or include it in our weekly wrap up thread. 
You're welcome to submit your tweet for amplification but please keep in mind if and how we interact with it is at our sole discretion.

### OPRadio

We also host a weekly Twitter Space in an interview/podcast format called OP Radio. You're welcome to express interest in being featured on OP Radio, but keep in mind that we maintain the final say on whether it makes sense or not.

 

> ## Grants Funding 🪙🪙

Reviewer: ```suggestion
## Governance Grants
```

> Grants are mainly handled by [The Optimism Collective](https://app.optimism.io/announcement), our open governance system/digital society. If you are interested in seeking funding here, we first recommend the following: 

Reviewer: ```suggestion
Grants are mainly handled by [The Optimism Collective](https://app.optimism.io/announcement), our open governance system/digital society. If you are interested in applying, we first recommend the following: 
```

::: tip Tip

> Grants are a lever to support sustainable growth in the ecosystem, therefore we do not provide grants for projects' operational expenses. 

Reviewer: Remove this, we are doing something along these lines for season 3. Instead:

Before applying, please check that your application falls within the possible Governance Grant categories in the [OPerating manual](https://github.com/ethereum-optimism/OPerating-manual/blob/main/manual.md#valid-proposal-types).

:::
 

- Deploy to Optimism (we are [EVM Equivalent](https://medium.com/ethereum-optimism/introducing-evm-equivalence-5c2021deb306), not just EVM compatible so it is very easy to deploy). 

>   Check out [our tutorial](https://github.com/ethereum-optimism/optimism-tutorial/tree/main/getting-started) to see how to deploy.

Reviewer: ```suggestion
  Check out our [getting started tutorial](https://github.com/ethereum-optimism/optimism-tutorial/tree/main/getting-started) to see how to deploy. We also have [tons of more in-depth tutorials](tutorials) for bridging and other L2 specific functionality. 
```

- Try gaining organic traction:
  - Traction can be gained through our community.
  - Head over to [our discord](https://discord-gateway.optimism.io/) and check out [#showandtell](https://discord.com/channels/667044843901681675/819333988719263774) or [#demoday](https://discord.com/channels/667044843901681675/960493820674465883).

>   Demoday is an AMA about your product to our community.

Reviewer: ```suggestion
  Demo Day is an AMA about your product to our community. Apply in Discord. 
```

  - Seek synergies – look at other teams already deployed on Optimism that could work with your product and reach out to them.
- Once you have done the above, check out our [governance grant process](https://gov.optimism.io/t/grant-proposal-template/3233).

> - To get reviews in real time from governance delegates go to discord and post about your product in [#temp-check](https://discord.com/channels/667044843901681675/1011238484373159956).

Reviewer: ```suggestion
- To get reviews in real time from governance delegates go to discord and post about your project in [#temp-check](https://discord.com/channels/667044843901681675/1011238484373159956).
```

 

## Other Helpful Links

Curious about our Optimistic Origins? 
Listen to [this OPRadio Episode](https://twitter.com/optimismFND/status/1585015947626426368) with one of our founders, [@JingleJam](https://twitter.com/jinglejamOP).

What do the next 100 years have in store for crypto? [@Karl_dot_tech](https://twitter.com/karl_dot_tech) shares his vision in [this chat with Bankless](https://www.youtube.com/watch?v=60F2wtjZO5M).

Want to see what we are doing with Retroactive Public Goods? 

> (Hint: We give all our sequencer ~100k/day in revenue, away to public goods so your users can rest easy knowing that when they use Optimism, they are contributing to a good cause). 

Reviewer: ```suggestion
(Hint: We give all our sequencer revenue (~100k/day!), away to public goods so your users can rest easy knowing that when they use Optimism, they are contributing to a good cause). 
```

> More on Public Goods Funding [here](https://app.optimism.io/announcement), [here](https://medium.com/ethereum-optimism/retroactive-public-goods-funding-33c9b7d00f0c), and [Vitalik’s](https://twitter.com/VitalikButerin) review of our first experimental round is [here](https://vitalik.ca/general/2021/11/16/retro1.html). 

Reviewer: Can we move this to its own page? Then add little one liners about each of these. We really need to have PGF as its own page under the citizen's house.

Author: Now that there's a page for it (https://community.optimism.io/docs/governance/retropgf-2/), can we just remove the material from here and link to it?

Author: If that is acceptable, just approve and merge - I changed it to that link and dealt with everything else.
